### ISSUE TITLE
[Request New Content] Cloud Hosting vs. Self Hosting

### ISSUE THREAD

**Issue by dessireugarte** [strapi/community-content#1080]
**My Request**

**Community vote**
👍 Upvote if you like this topic
🚀 If you can make the article/video!

Thank you.

**Comment by danielonugha0** [comment]
Hi @dessireugarte @PaulBratslavsky 
 I would love to write on this topic
Best regards

**Comment by danielonugha0** [comment]
Hi @PaulBratslavsky @dessireugarte 
Here is the outline to the topic

Comparing Cloud Hosting Vs. Self Hosting
If you’re reading this article on a phone or desktop, you have just used several cloud-based applications while trying to get to see this article.
Cloud hosting is when an organization installs and accesses software on a server over the internet that is owned and managed by a third-party organization. Self-hosting is when an organization installs and accesses software from its server. Self-hosting is also often referred to as on-premises or local hosting.

Goal
In this article, we will zoom in on two popular software programs: Cloud Hosting vs. Self Hosting.

Table of Content

- Introduction
- what is cloud Hosting?
    - Uses of Cloud Hosting
    - Advantages of Cloud Hosting
    - Disadvantages of Cloud Hosting
    - Kinds of Cloud Hosting
- What is Self Hosting?
    - Advantages of Self Hosting
    - Disadvantages of Self Hosting
    - Uses of Self Hosting
- Comparison between Cloud Hosting and Self Hosting
- Why Cloud Hosting is the best
- Conclusion

**Comment by Xemarij** [comment]
Hi @dessireugarte and @PaulBratslavsky,
I am interested in the topic.
Please see my writing portfolio here: https://zemaarij.contently.com/

I eat, drink and sleep writing. I am attaching one of my star articles on a similar topic as a writing skill evaluation sample, I wrote for a client who loves it.

[20 Wix competitors.edited.docx](https://github.com/strapi/community-content/files/10306922/20.Wix.competitors.edited.docx)

**Comment by Charity-Mutembei** [comment]
Hello, @dessireugarte @PaulBratslavsky 
I would love to take on this task as well. Below is a proposed outline for the article in the discussion. I have considered those at the beginner level on the topic and those who advanced through the outline and sections. 

I. Introduction

Definition of cloud hosting and self-hosting
Overview of the main differences between the two

II. Cloud Hosting

Definition and explanation of cloud hosting
Advantages of cloud hosting:
Scalability
Reliability
Cost-effectiveness
Convenience
Disadvantages of cloud hosting:
Limited control over the hosting environment
Dependence on a third-party provider
Potential for additional costs for high usage

III. Self-Hosting

Definition and explanation of self-hosting
Advantages of self-hosting:
Greater control over the hosting environment
Potential for cost-effectiveness
Ability to customize the server to meet specific needs
Disadvantages of self-hosting:
Requires technical expertise and resources to manage
Responsibility for maintenance, updates, backups, and security measures
Limited scalability without physical changes to the hosting environment

IV. Comparison of Cloud Hosting and Self-Hosting

Summary of the main differences between cloud hosting and self-hosting
Factors to consider when deciding between the two
Examples of when one option may be more suitable than the other

**Comment by robinokwanma** [comment]
Hi @PaulBratslavsky  @dessireugarte 
I would love to write on this topic.

After a little research i noticed strapi is about to launch "strapi cloud" as an alternative to the self hosted option. I'll also love to write about both options in the article

**Comment by ibn-ashimi** [comment]
Hi, @dessireugarte Considering I have written quite a few articles around hosting with strapi, I would really love to work on this too. Familiar Concepts.

**Comment by hubertnare** [comment]
Hello @dessireugarte  and @PaulBratslavsky,

I hope I find you well.

I intend to write the article “Cloud hosting vs. self-hosting."

While self-hosting may not necessarily imply “on-premises” or “local-hosting” but it is all about having full control over one’s data or software at the application or OS level. In other words, one can choose to self-host using a VPS. Essentially, a server one can fully claim control over. This can be achieved through virtualization, VPS, and sometimes dedicated servers.

Or one can use literal, cool-looking servers with blinking lights, stacked on a server rack, one atop each other, located in a home lab or an organizational room.

Both approaches lead to self-hosting. But, in this article, I will be looking at VPSs and self-hosting.

Below is my outline.

**Comment by hubertnare** [comment]
**Abstract:**

A web hosting service enables individuals and organizations to make their websites available on the internet. There are a plethora of web hosting services, each of which offers a different set of features and degrees of performance, including shared hosting, dedicated hosting, and cloud hosting. The article’s aim, regarding web hosting, is to pay close attention to cloud hosting, which is a rule in the tech sector at the moment, and self-hosting, a tech-savvy path that is seldom trodden upon (at least by most individuals). By contrast, it can also help you decide which one is better.

**Outline:**

**Introduction**

**Web hosting at a glance**

- What is web hosting?
- Internet hosting service
- Going online: from local to live
- types of web hosting

**What is self-hosting?**

- What is self-hosting?
- You don’t need rack of stacking servers to self-host
- Why self-host?
- what is a VPS?
- what is a Dedicated server?
- how to self-host

**What is Cloud hosting?**

- What is cloud hosting?
- Why cloud host?
- different types of cloud hosting
- how to cloud-host
- comparing different cloud hosts
- choosing a cloud host provider

**Cloud hosting Vs. Self hosting**

- cost
- control
- time to market
- technical know-how (responsibility)
- maintenance
- scalability
- Security

**Decide which option is best suited for you?**

**Alternative cloud host -  Strapi Cloud**

- Quick and easy deployment
- CDNs
- Backups
- Multidatabase support

**Conclusion**

- wrap up

**Comment by quentin-js** [comment]
Hi @dessireugarte, hi @Mcastres hi @PaulBratslavsky! I hope you folks are fine!

I had some thoughts during my morning running this Saturday (true story :D) and I'd like to suggest a slight change to the main topic? Here is my proposal: it exists many hostings approach for CMS and the point is that each one is good for a specific persona or use case. As Strapi tends to be closer to a PaaS, I would even make sense to talk about Platform Engineer as it is the latest trend in this area.

As a result, I would the content as the following:

**Introduction**: a quick reminder of the positioning of headless CMS vs traditional CMS (this is important for what's next)

**Par* I*: the ideal hosting solution for business users => I'll mainly talk about the SaaS approach and why it benefits more to business personas more, usually not able to rely on developers...
--- transitioning by explaining that most enterprise-grade companies have a dedicated dev team (internal or external)

**Part II**: the ideal hosting solution for sensitive industries => I'll introduce the benefits of self-hosting and why it is an even greater idea for sensitive businesses (banking, healthcare...)
--- transitioning by explaining as self-hosting is also a great first step IF you have an existing infrastructure setup but not always the case...

**Part III**: the ideal solution for developers => I'd like to talk about PaaS and in particular, Platform Engineering as this is a more and more trendy concept

**Conclusion**: I would end with a focus on Strapi Cloud and reuse my beta tester experience


Happy to have your feedback! Cya!

**Comment by danielonugha0** [comment]
Hi @PaulBratslavsky @Mcastres 
Can I go ahead with the topic?

**Comment by danielonugha0** [comment]
@PaulBratslavsky @Mcastres @hubertnare 
Can I proceed?

**Comment by danielonugha0** [comment]
Hi @PaulBratslavsky @Mcastres @hubertnare 

I have been waiting to hear from you all, if I should proceed and write the article

**Comment by PaulBratslavsky** [comment]
@danielonugha0 sorry for the confusion.  I assigned this to you and marked `in progress`.  It was my fault I should of been more clear.
![Screenshot 2023-03-10 at 3 54 52 PM](https://user-images.githubusercontent.com/6153188/224435779-cecfd68f-8e3e-4838-a2c0-808f8abe626e.png)

**Comment by danielonugha0** [comment]
Hi @PaulBratslavsky I won't love to write on this topic.

**Comment by danielonugha0** [comment]
Hi @PaulBratslavsky 
I will start working on the article.

**Comment by vcoisne** [comment]
@danielonugha0 please do not work on this article. We already have a very similar article https://strapi.io/blog/on-premise-vs-cloud-solutions-which-is-the-best-for-your-business and are not accepting new WFC article at this time.

**Comment by PaulBratslavsky** [comment]
@vcoisne thank you for catching this.  I will close this issue due to similar content.

**Comment by hubertnare** [comment]
Well, this was one heck of a confusing issue. My apologies if I contributed to it. Apparently I worked on the article on the due to the assignment but somehow forgot to share the link to the draft. But, no worries!

However, I will take responsibility for this oversight and self-publish it on my personal blog. I hope this will not be a major issue.
 
Does this work for you? @vcoisne  , @PaulBratslavsky  & of course @dessireugarte  .
